specification/redisenterprise/resource-manager/Microsoft.Cache/stable/2021-03-01/redisenterprise.json

```diff
@@ -5,7 +5,7 @@
     "description": "REST API for managing Redis Enterprise resources in Azure.",
     "version": "2021-03-01",
     "x-ms-code-generation-settings": {
-      "name": "redisenterpriseClient"
+      "name": "redisEnterpriseClient"
```

Comment:
Shouldn't the initial R be upper-case too since that is the usual style for class names?

Reply:
Yes